### PR TITLE
Bare metal provisioning documentation

### PR DESCRIPTION
**Pull request by matejart**
This improves on the documentation for CF 4.7. We described the bare metal provisioning.

### CODE REVIEW

File: redfish_guide/topics/using-redfish-physical-infra.adoc [hunk 1]
```diff
@@ -126,3 +124,67 @@ Again depending on the {redfish} implementation, the statuses will cycle from
...
+*Lifecycle* menu.
+
+image::docs_redfish_using15_lifecycle_provisioning_menu.png[alt="Lifecycle menu for Provisioning Selected Physical Servers",350,align="center"]
+
+Click the *Lifecycle -> Provision Selected Physical Servers*. This opens the
+dialog for filling in the request.
+
+image::docs_redfish_using16_provisioning_dialog_default.png[alt="Provision Selected Physical Servers dialog",350,align="center"]
+
+NOTE: If from the server list, we click on a server, the we can initiate the
```
Comment (miha-plesko) [inline redfish_guide/topics/using-redfish-physical-infra.adoc]: the -> then

File: redfish_guide/topics/using-redfish-physical-infra.adoc [hunk 2]
```diff
@@ -126,3 +124,67 @@ Again depending on the {redfish} implementation, the statuses will cycle from
...
+same dialog by clicking *Lifecycle -> Provision Physical Server* from the
+Physical Server's summary
```
Comment (miha-plesko) [inline redfish_guide/topics/using-redfish-physical-infra.adoc]: End sentence with dot

File: redfish_guide/topics/using-redfish-physical-infra.adoc [hunk 3]
```diff
@@ -126,3 +124,67 @@ Again depending on the {redfish} implementation, the statuses will cycle from
 Please refer to the <<ReleaseNotesRedfish>> section for further information
 about the power management actions and statuses.
 
+=== Physical Server Provisioning
+{redfish-provider-full} lets the users request to initiate network booting of
+an Operating System.
```
Comment (miha-plesko) [inline redfish_guide/topics/using-redfish-physical-infra.adoc]: It feels like we only tell first half of the story. I would reword it a little to emphasize that we allow for OS installment, not just network boot. Something like:

_{redfish-provider-full} allows user to install operating system on physical server by leveraging PXE network boot mechanism. Basically, what happens is that physical server is rebooted with "boot from LAN" option set while PXE server is connected to the same LAN and pre-configured to install selected operating system on that specific physical server. Steps below demonstrate how to initialize provisioning for one or more physical servers._

File: redfish_guide/topics/using-redfish-physical-infra.adoc [hunk 4]
```diff
@@ -126,3 +124,67 @@ Again depending on the {redfish} implementation, the statuses will cycle from
 Please refer to the <<ReleaseNotesRedfish>> section for further information
 about the power management actions and statuses.
 
+=== Physical Server Provisioning
```
Comment (miha-plesko) [inline redfish_guide/topics/using-redfish-physical-infra.adoc:127]: Perhaps a note would be in order to warn user she needs to have PXE server configured upfront and that PXE configuration is not really part of this documentation.

**Review by miha-plesko — APPROVED** [review]
I like it. Couple of small comments but in general LGTM. In the main "Physical Server Provisioning" chapter I'd repeat some things that you already mention in prerequisites and supported features section (see comments) because most users will probably just jump to that chapter (I would, too, probably :=).

File: redfish_guide/topics/using-redfish-physical-infra.adoc (hunk 1 above)
Comment (matejart) [reply redfish_guide/topics/using-redfish-physical-infra.adoc]: 👍

File: redfish_guide/topics/using-redfish-physical-infra.adoc (hunk 2 above)
Comment (matejart) [reply redfish_guide/topics/using-redfish-physical-infra.adoc]: 👍

File: redfish_guide/topics/using-redfish-physical-infra.adoc (hunk 3 above)
Comment (matejart) [reply redfish_guide/topics/using-redfish-physical-infra.adoc]: 👍 
I worded it differently, but in the suggested spirit.

File: redfish_guide/topics/using-redfish-physical-infra.adoc (hunk 4 above)
Comment (matejart) [reply redfish_guide/topics/using-redfish-physical-infra.adoc:127]: 👍 
Warning them implicitly.

**Review by miha-plesko — APPROVED** [review]
LGTM

**Review by tadeboro — APPROVED** [review]
I read through the whole document and found no issues. Thank you for our effort @matejart
